frontend/app/src/pages/main/v1/workflows/$workflow/components/trigger-workflow-form.tsx

```diff
@@ -241,28 +281,6 @@ export function TriggerWorkflowForm({
     }
   };
 
-  if ((!workflowKeys || workflowKeys.rows?.length === 0) && isFetched) {
-    return (
-      <Dialog
-        open={show}
-        onOpenChange={(open) => {
-          if (!open) {
-            handleClose();
-          }
-        }}
-      >
-        <DialogContent className="sm:max-w-[625px] py-12 max-h-screen overflow-auto">
-          <DialogHeader className="gap-2">
-            <DialogTitle>Trigger Workflow</DialogTitle>
-            <DialogDescription className="text-muted-foreground">
-              No workflows found. Create a workflow first.
-            </DialogDescription>
-          </DialogHeader>
-        </DialogContent>
-      </Dialog>
-    );
-  }
-
```

Comment:
I think we might still need this? maybe I'm misunderstanding the intention here, but this is the empty state in the trigger modal right?